CHANGELOG.rst

```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
+-------------
+
+- Added search_child_modules option to terraform_provider inventory plugin.
+- Allow project_path in terraform_provider inventory plugin to accept a list of paths.
+
+Minor Changes
+-------------
+
+- Removed integration tests workaround in terrform_provider. (https://github.com/ansible-collections/cloud.terraform/pull/84)
+- Set default of state_file in terraform_provider inventory plugin to a blank string.
+- add support for workspace when running `terraform output` in modules `cloud.terraform.terraform`, `cloud.terraform.terraform_output` and lookup `cloud.terraform.tf_output` (https://github.com/ansible-collections/cloud.terraform/issues/85).
```

Comment:
We should also update version_added for this option https://github.com/ansible-collections/cloud.terraform/pull/88/files#diff-fedb790d9a5867b094724ba9d63a33283da12b08bbfe9123f121d805a14c541eR39.
```suggestion
- terraform  - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85). 
- terraform_output -  add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
- tf_output - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
```